README.md

```diff
@@ -3,11 +3,23 @@
 [![Build Status](https://travis-ci.org/rdf-ext/rdf-ext.svg?branch=master)](https://travis-ci.org/rdf-ext/rdf-ext)
 [![npm version](https://badge.fury.io/js/rdf-ext.svg)](https://badge.fury.io/js/rdf-ext)
 
-RDF-Ext provides a JavaScript library for working with RDF & Linked Data.
+RDF-Ext provides a low-level JavaScript library for working with RDF & Linked Data.
 This module contains the core classes to handle RDF Model data.
 Additional modules may be required to handle data in different formats (Turtle, JSON-LD) or stores (Web, SPARQL).
 The [module section](#modules) lists the most common modules.
 
+By including this package, you will get an implementation of:
+
+* The [RDF/JS Datamodel specification](http://rdf.js.org/data-model-spec/), provided by [@rdfjs/data-model](https://www.npmjs.com/package/@rdfjs/data-model)
+* A fast implementation of the [RDF/JS Dataset specification](https://rdf.js.org/dataset-spec/), provided by [rdf-dataset-indexed](https://www.npmjs.com/package/rdf-dataset-indexed)
+* A simple [N-Triples Serializer](https://en.wikipedia.org/wiki/N-Triples), provided by [@rdfjs/to-ntriples](https://www.npmjs.com/package/@rdfjs/to-ntriples)
+* A [RDF Dataset Normalization](https://json-ld.github.io/normalization/spec/), provided by [rdf-normalize](https://www.npmjs.com/package/rdf-normalize)
+
+This should be all needed for starting to work with RDF in JavaScript.
+If you interact with [named nodes](https://rdf.js.org/data-model-spec/#namednode-interface) directly we recommend [namespace](https://github.com/rdfjs-base/namespace), which gives a nice interface for creating named nodes. RFD-ext provides a package with [commonly used namespaces](https://github.com/rdf-ext/rdf-namespaces-common) prepopulated.
```

Comment:
Just remove sentence about `rdf-namespaces-common`, cause I just archived the package. The package is way to big and needs a concept to make it smaller. That's not high prio, so I can take a while till it will be unarchived again.